omegaconf/dictconfig.py

```diff
@@ -401,14 +400,18 @@ def __delitem__(self, key: DictKeyType) -> None:
                 ),
             )
 
-        del self.__dict__["_content"][key]
-
-    def get(self, key: DictKeyType, default_value: Any = DEFAULT_VALUE_MARKER) -> Any:
+        try:
+            del self.__dict__["_content"][key]
+        except KeyError:
+            msg = "Key not found: '$KEY'"
+            self._format_and_raise(key=key, value=None, cause=ConfigKeyError(msg))
+
+    def get(self, key: DictKeyType, default_value: Any = None) -> Any:
```

Comment:
can you rebase on the other diff and remove the get changes from this one?

Reply:
Done.